src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (callback) {
+    if (!callback) {
+      /* eslint-disable no-param-reassign */
+      callback = (a, b) => {
+        a = String(a);
+        b = String(b);
+
+        switch (true) {
+          case a > b:
+            return 1;
+
+          case a < b:
+            return -1;
+
+          default:
+            return 0;
+        }
+      };
+    }
```

Comment:
just add the default value for callback instead of this 
```suggestion
  [].__proto__.sort2 = function (callback = (a,b) => {...}) {
```